bee-storage/bee-storage-rocksdb/src/storage.rs

```diff
@@ -42,7 +42,12 @@ impl Storage {
 
         let cf_message_id_to_message = ColumnFamilyDescriptor::new(CF_MESSAGE_ID_TO_MESSAGE, Options::default());
 
-        let cf_message_id_to_metadata = ColumnFamilyDescriptor::new(CF_MESSAGE_ID_TO_METADATA, Options::default());
+        fn keep_current(_key: &[u8], existing_val: Option<&[u8]>, operands: &MergeOperands) -> Option<Vec<u8>> {
+            existing_val.or_else(|| operands.into_iter().next()).map(|v| v.to_vec())
```

Comment:
What does `operands.into_iter().next()` do exactly ?

Reply:
takes the first operand passed to `merge_cf` in case the value does not exist.